### DISCUSSION TITLE
Dbeaver Importing Templates Not Working for Enterprise Edition

### DISCUSSION THREAD

**Discussion by siracdemir** [dbeaver/dbeaver#19527] [Q&A] [answered]
Hi,

I was using Dbeaver community.
Recentyl, I have been using Enterprise Edition.
I exported templates from Dbeaver comminity.
I have imported that exported template to Enterprise Edition, but nothing appears. 
Almost all of my template snippets are SQL and SQL Server context.

Please , you can see a very simple exported template below(templates_2023-03-29.zip)
Note: my Dbeaver community version: 23.0.0.202303040621
Enterprise Edition version: 23.0.0.202303131136

[templates_2023-03-29.zip](https://github.com/dbeaver/dbeaver/files/11097587/templates_2023-03-29.zip)

**Comment by LonwoLonwo** [comment]
Hello @siracdemir 

Please create a ticket on our tech support for the Enterprise users: https://dbeaver.com/signin/

**Comment by siracdemir** [comment, marked as answer]
Hi,

I found the problem. 
When you export file from community edition the name of context is :
context="sql_sqlserver_microsoft"
You need to rename context to sql_sqlserver-ee_microsoft

As an example:
If you import file to community  edition, the xml should look like this:
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<templates>
	<template autoinsert="true" context="sql_sqlserver_microsoft" deleted="false" description="" enabled="true" name="sfrom">select * from dbo.${table}</template>
</templates>


If you import file to enterprise edition, the xml should look like this:
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<templates>
	<template autoinsert="true" context="sql_sqlserver-ee_microsoft" deleted="false" description="" enabled="true" name="sfrom">select * from dbo.${table}</template>
</templates>
